code/datums/sexcon/sex_actions/deviant/force_thighjob.dm

```diff
@@ -6,6 +6,8 @@
 		return FALSE
 	if(!target.getorganslot(ORGAN_SLOT_PENIS))
 		return FALSE
+	//if(HAS_TRAIT(user, TRAIT_TINY)) //Fairy is too small and weak to force this
+	//	return FALSE
```

Comment:
Remove these lines if they're not meant to be kept in there

Reply:
Removed